src/utils/index.js

```diff
@@ -54,7 +54,7 @@ export const TOO_MANY_FILES_REJECTION = {
 // that MIME type will always be accepted
 export function fileAccepted(file, accept) {
   const isAcceptable =
-    file.type === "application/x-moz-file" || accepts(file, accept);
+    file.type === "application/x-moz-file" || file.type === '' || accepts(file, accept);
```

Comment:
I'm not too sure about this change and how it will affect the current behaviour. I'm trying to think if there are any other instances where the type may be missing but it's still a file and not a directory.

Also, the function is applied to files, not directories, so I'm a bit confused 🤔  Maybe you could provide some screencast of the issue you're trying to solve here.